### PR TITLE
finance_crew

### PR DESCRIPTION
**Pull request by maximiliangarmatsch**

### CODE REVIEW

File: finance_crew/src/finance_crew/config/agents.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+admin_assistant_manager:
+  role: >
+    Admin Assistant Manager
+  goal: >
+    Uncover insights into the bank account report. For this Search through the PDF and give
```
Comment (maximiliangarmatsch) [inline finance_crew/src/finance_crew/config/agents.yaml]: run through spell/grammer checker

File: finance_crew/src/finance_crew/config/tasks.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+admin_assistant_manager_task:
```
Comment (maximiliangarmatsch) [inline finance_crew/src/finance_crew/config/tasks.yaml]: 1. bad naming. an admin_assistant_manager will most likely have multiple tasks.
2. i think tasks should be more atomic
please split this up and come up with better naming

File: finance_crew/src/finance_crew/config/tasks.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    
+    What is our plan regarding the investments  mentioned in the Assistant Admin Manager report?
+    
+    How much money do we need to investment give all options including key points for each?
+    
+    Do we have enough money to invest in plan approved with CEO?
+    
+    What is the strategy to get a loan for more money?
+    
+    Provide a detailed analysis on each of the questions. And also get approved each questions with by the CEO to take furthre steps. If required
```
Comment (maximiliangarmatsch) [inline finance_crew/src/finance_crew/config/tasks.yaml]: spell/grammer check

File: finance_crew/src/finance_crew/config/tasks.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    
+    Provide a detailed analysis on each of the questions. And also get approved each questions with by the CEO to take furthre steps. If required 
+    loan for investment do internet search for best avialble loan plan. 
+  expected_output: >
+    Analysis report with deeper insights like a real human capabilities.
+
+chief_excutive_officer_task:
+  description: >
+    Using the insights provided by the Assitant Admin Manager and Chief Finance Officer, please craft an expertly
+    styled decision that is targeted towards the investor community. Make sure to also include the long-term implications
```
Comment (maximiliangarmatsch) [inline finance_crew/src/finance_crew/config/tasks.yaml]: what is "investor community"?

File: finance_crew/src/finance_crew/config/tasks.yaml [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    of the insights that are mentioned.  
+    
+    Please ensure that you are the CEO of the company you may approved the Chief Finnace Officer decision or also ask for chnages with
```
Comment (maximiliangarmatsch) [inline finance_crew/src/finance_crew/config/tasks.yaml]: spellcheck and its not quite clear what you mean here

File: finance_crew/src/finance_crew/config/tasks.yaml [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+    directed guidelines.
```
Comment (maximiliangarmatsch) [inline finance_crew/src/finance_crew/config/tasks.yaml]: what are directed guidelines?

File: finance_crew/src/finance_crew/crew.py [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+from crewai.project import CrewBase, agent, crew, task
+from crewai_tools import SerperDevTool, ScrapeWebsiteTool, PDFSearchTool
+
+pdf_search_tool = PDFSearchTool(
+    pdf = "/home/aamir/AI-agents-video-main/Crewai examples latest/crewAI-examples/finance_crew/src/finance_crew/report.pdf",
+)
+
+@CrewBase
+class FinnaceCrew():
+	"""MarketingPosts crew"""
```
Comment (maximiliangarmatsch) [inline finance_crew/src/finance_crew/crew.py]: type?

File: finance_crew/trained_agents_data.pkl
Comment (maximiliangarmatsch) [inline finance_crew/trained_agents_data.pkl:1]: if this is a temp file, then see if you can move it to a sub folder /temp

File: finance_crew/src/finance_crew/report.pdf
Comment (maximiliangarmatsch) [inline finance_crew/src/finance_crew/report.pdf:1]: move to /assets

File: finance_crew/src/finance_crew/config/agents.yaml (hunk 1 above)
Comment (USTAADCOM) [reply finance_crew/src/finance_crew/config/agents.yaml]: ok

File: finance_crew/src/finance_crew/config/tasks.yaml (hunk 2 above)
Comment (USTAADCOM) [reply finance_crew/src/finance_crew/config/tasks.yaml]: Ok

File: finance_crew/src/finance_crew/config/tasks.yaml (hunk 3 above)
Comment (USTAADCOM) [reply finance_crew/src/finance_crew/config/tasks.yaml]: ok

File: finance_crew/src/finance_crew/config/tasks.yaml (hunk 4 above)
Comment (USTAADCOM) [reply finance_crew/src/finance_crew/config/tasks.yaml]: ok

File: finance_crew/src/finance_crew/config/tasks.yaml (hunk 5 above)
Comment (USTAADCOM) [reply finance_crew/src/finance_crew/config/tasks.yaml]: ok

File: finance_crew/src/finance_crew/config/tasks.yaml (hunk 6 above)
Comment (USTAADCOM) [reply finance_crew/src/finance_crew/config/tasks.yaml]: ok

File: finance_crew/src/finance_crew/crew.py (hunk 7 above)
Comment (USTAADCOM) [reply finance_crew/src/finance_crew/crew.py]: ok

File: finance_crew/src/finance_crew/report.pdf
Comment (USTAADCOM) [reply finance_crew/src/finance_crew/report.pdf:1]: ok

File: finance_crew/trained_agents_data.pkl
Comment (USTAADCOM) [reply finance_crew/trained_agents_data.pkl:1]: Crew manage this path automaticlly.

**Pull request merged**